src/bootstrap/config.rs

```diff
@@ -411,7 +411,11 @@ pub struct Target {
 impl Target {
     pub fn from_triple(triple: &str) -> Self {
         let mut target: Self = Default::default();
-        if triple.contains("-none") || triple.contains("nvptx") || triple.contains("switch") {
+        if triple.contains("-none")
+            || triple.contains("nvptx")
+            || triple.contains("switch")
+            || triple.contains("-uefi")
```

Comment:
I don't recognize these other targets. "switch" is the Nintendo switch? What's nvptx? How did you determine these don't have std available?

Reply:
Those were already there, rustfmt just put them on different lines due to the single line getting too long.

Reply:
Ugh, rustfmt strikes again. Thanks.